ci-operator/config/openshift/installer/openshift-installer-release-4.19.yaml

```diff
@@ -1140,6 +1140,19 @@ tests:
   steps:
     cluster_profile: openstack-hwoffload
     workflow: openshift-e2e-openstack-dualstack
+- always_run: false
+  as: e2e-vsphere-ovn-dualstack
+  optional: true
+  steps:
+    cluster_profile: vsphere-elastic
+    env:
+      EXTRA_MG_ARGS: --host-network
+      TEST_SKIPS: Services Secondary IP Family \[LinuxOnly\] should function for pod-Service\|
+        Services Secondary IP Family \[LinuxOnly\] should function for node-Service\|
+        Services Secondary IP Family \[LinuxOnly\] should function for endpoint-Service\|
+        Services Secondary IP Family \[LinuxOnly\] should function for service endpoints
+        using hostNetwork\| should have ipv4 and ipv6 internal node ip
```

Comment:
We shouldn't skip those tests any more. I know they are still skipped in CNO because it's impossible to get a quick approval there, but starting from around 4.15 or 4.16 those tests should actually pass